programs/ssl/ssl_test_common_source.c

```diff
@@ -159,69 +159,6 @@ int dtls_srtp_key_derivation( void *p_expkey,
 
 #endif /* MBEDTLS_SSL_EXPORT_KEYS */
 
-#if defined(MBEDTLS_SSL_RECORD_CHECKING)
-int ssl_check_record( mbedtls_ssl_context const *ssl,
```

Comment:
Same as the other record checking block: when resolving conflicts we want to keep this block.

Reply:
Done